setup.py

```diff
@@ -0,0 +1,22 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="torch-expm-Lezcano", # Replace with your own username
```

Comment:
Probably calling it `pytorch-expm` would be better. Going with this, the directory should be called `pytorch-expm`.

Reply:
Directory is now called `pytorch_expm` with the package being called `pytorch-expm`, since that's what's compatible with import statements in Python.